README.md

```diff
@@ -1 +1,11 @@
-# angular-native-dom-apis-lab
\ No newline at end of file
+# Native DOM APIs
+
+## Objectives
+
+- Use a native event API
+- Use a Controller inside Directive
+- Update Controller layer using native events
+
+## Instructions
+
+Using the directive in this repo, add event listeners to change the text when the user clicks. Remember to use `require` and `$apply()`!
```

Comment:
tell the student exactly what the text should update to ie "When the user clicks the span, update the text to say 'Thank you For Clicking!'"